packages/gasket-plugin-swagger/package.json

```diff
@@ -51,7 +51,7 @@
     "eslint-plugin-jest": "^27.2.1",
     "eslint-plugin-json": "^3.1.0",
     "eslint-plugin-unicorn": "^44.0.0",
-    "fastify": "^3.0.0",
+    "fastify": "^4.0.0",
```

Comment:
Shouldn't we leave this on fastify 3 for now?

Reply:
Yes, sorry don't know how that slipped in there